HackYourFuture-CPH/fp-class22
# Add eslint ignore for client

# Description

Without this fix `/packages/client/build` would get processed by eslint.

# How to test?

```
cd packages/server
yarn build

cd ../..
yarn validate
```

Should not throw any eslint errors on files in the client build.


# Checklist
